src/player.js

```diff
@@ -1653,6 +1669,11 @@ export default class Player extends FakeEventTarget {
           return this.dispatchEvent(event);
         });
       });
+      this._eventManager.listen(this._engine, Html5EventType.SEEKING, () => {
+        if (this.isLive()) {
+          this._isOnLiveEdge = this.duration && this.currentTime ? this.currentTime >= this.duration - 1 && !this.paused : false;
```

Comment:
-1  - is for the buffer size where the liveEdge could be on?

Reply:
It should be almost the same. there is a small difference because we reduce 0.1 from the currentTime if it greater or equal to duration

Reply:
So I think it should be on meaningful const